helios-services/src/main/java/com/spotify/helios/agent/SyslogRedirectingContainerDecorator.java

```diff
@@ -83,4 +101,20 @@ public void decorateContainerConfig(Job job, ImageInfo imageInfo,
     volumes.add("/helios");
     containerConfig.volumes(volumes);
   }
+
+  private boolean useSyslogRedirector(Optional<String> dockerVersion) {
+    if (!dockerVersion.isPresent()) {
+      // always use syslog-redirector if we can't figure out the version
+      return true;
+    } else {
+      final String version = dockerVersion.get();
+
+      if (version.startsWith("1.6.") || version.startsWith("1.7.") || version.startsWith("1.8.")) {
```

Comment:
We have `PomVersion.compareTo()` which compares semantic version strings. Perhaps use this class and rename it instead of using three specific hardcoded versions?